docs/content/docs/ops/state/state_backends.md

```diff
@@ -193,7 +193,7 @@ While we encourage the use of incremental checkpoints for large state, you need
   - Setting a default in your [Flink configuration file]({{< ref "docs/deployment/config#flink-configuration-file" >}}): `execution.checkpointing.incremental: true` will enable incremental checkpoints, unless the application overrides this setting in the code.
   - You can alternatively configure this directly in the code (overrides the config default): `EmbeddedRocksDBStateBackend backend = new EmbeddedRocksDBStateBackend(true);`
 
-Notice that once incremental checkpoont is enabled, the `Checkpointed Data Size` showed in web UI only represents the 
+Notice that once incremental checkpoint is enabled, the `Checkpointed Data Size` showed in web UI only represents the 
```

Comment:
we should track the chinese change - which on the face of it (from google translate) says incremental snapshots not checkpoints.